packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -976,6 +976,16 @@ class WebpackCLI implements IWebpackCLI {
         description: "Stop watching when stdin stream has ended.",
         negatedDescription: "Do not stop watching when stdin stream has ended.",
       },
+      {
+        name: "fail-on-warnings",
+        configs: [
+          {
+            type: "boolean",
```

Comment:
Because we don't want `--no-fail-on-warnings`
```suggestion
            type: "enum",
            values: [false],
```